neurons/validator.py

```diff
@@ -800,13 +801,43 @@ async def run_step(self):
 
         # Compute wins and win rates per uid.
         wins, win_rate = pt.validation.compute_wins(
-            uids, losses_per_uid, batches, uid_to_block
+            uids, losses_per_uid, batches, uid_to_block, constants.timestamp_epsilon
         )
 
         # Compute softmaxed weights based on win rate.
         model_weights = torch.tensor(
             [win_rate[uid] for uid in uids], dtype=torch.float32
         )
+
+        # If we are running the epsilon experiment for competition 1 then also try the experiment epsilon.
+        if (
+            competition.id == CompetitionId.B7_MODEL
+            and cur_block >= constants.timestamp_epsilon_experiment_start_block
+        ):
+            wins_epsilon_experiment, win_rate_epsilon_experiment = (
+                pt.validation.compute_wins(
+                    uids,
+                    losses_per_uid,
+                    batches,
+                    uid_to_block,
+                    constants.timestamp_epsilon_experiment,
+                )
+            )
+
+            # Overwrite model weights using a ratio between regular and experiment win rates.
```

Comment:
Note. We actually should overwrite the step weights instead with this ratio as otherwise the temperature will crush it.